backend/src/BuiltinCliHost/Libs/Cli.fs

```diff
@@ -298,8 +302,11 @@ let fns : List<BuiltInFn> =
                       |> DString
                       |> resultError
                   | Ok value ->
-                    let asString = LibExecution.DvalReprDeveloper.toRepr value
-                    return resultOk (DString asString)
+                    match value with
+                    | DString s -> return resultOk (DString s)
+                    | _ ->
```

Comment:
DvalReprDeveloper.toRepr wraps strings in quotes - so this would result in double-encoded DStrings when calling `Cli.runFunction` on fns that return strings